docs/guides/src/main/server/configuration.adoc

```diff
@@ -4,202 +4,237 @@
 
 <@tmpl.guide
 title="Configuring Keycloak"
-summary="An overview of the server configuration">
+summary="Understand how to configure and start Keycloak">
 
-This guide describes underlying core concepts of Keycloak configuration. It includes configuration guidelines for optimizing Keycloak for faster startup and low memory footprint.
+This guide explains the configuration methods for Keycloak and how to start and apply the preferred configuration. It includes configuration guidelines for optimizing Keycloak for faster startup and low memory footprint.
 
-== Server options
+== Configuration Sources for Keycloak
+Keycloak loads the configuration from four different configuration sources:
 
-You can view the server options by adding the `help` option for individual commands:
+* command-line parameters
+* environment variables
+* user-created `keycloak.conf` file
```

Comment:
Maybe add in which directory?

Reply:
you can choose freely for that one, so not really sure what to write here. You have a suggestion? :) But i'll change it to `*.conf` bc it is not needed to be named keycloak.conf

Reply:
Maybe its also not that important. It just came to my mind because the next option specified a path. If the path is arbitrary, maybe rather leave it out.